### DISCUSSION TITLE
Addition template

### DISCUSSION THREAD

**Discussion by secondl1ght** [dennisreimann/bitcoin-donations#118] [Q&A] [answered]
Is there an addition template showing all the options available in the JSON when adding a new entry? Or where in the code can I view the available options?

**Comment by dennisreimann** [comment, marked as answer]
Hey, great idea! Added it in 6e17d5e16c78b9beeaa7bc9a1de7df3fd3af1780. Thanks!

**Reply by secondl1ght** [reply]
Awesome thanks!
